pkgs/applications/networking/geph5/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libxkbcommon,
+  openssl,
+  jemalloc,
+  sqlite,
+  vulkan-loader,
+  wayland,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "geph5";
+  version = "0.2.55";
+
+  src = fetchFromGitHub {
+    owner = "geph-official";
+    repo = "geph5";
+    rev = "geph5-client-v${finalAttrs.version}";
+    hash = "sha256-oq/aPh60QtBQ+tdX7j1NaCCe8G5THsrqAd4Is4yc9fA=";
+  };
+
+  cargoHash = "sha256-jzjCTQRhUr2Eif6/AzZCZSrKjNbJdR60qgZ6xQ87B1w=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    libxkbcommon
+    openssl
+    jemalloc
+    sqlite
+    vulkan-loader
+    wayland
+  ];
+
```

Comment:
This package may require patch to ensure runtime dependencies (e.g., `ip` and `iptables` commands) are available in nix: 
https://github.com/geph-official/geph5/blob/d793aa89e862e11464c46f5cc0ab1b6011215f07/binaries/geph5-client/src/vpn/linux.rs#L33
https://github.com/geph-official/geph5/blob/d793aa89e862e11464c46f5cc0ab1b6011215f07/binaries/geph5-client/src/vpn/linux_routing_setup.sh#L1

Reply:
> > This package may require patch to ensure runtime dependencies (e.g., `ip` and `iptables` commands) are available in nix: https://github.com/geph-official/geph5/blob/d793aa89e862e11464c46f5cc0ab1b6011215f07/binaries/geph5-client/src/vpn/linux.rs#L33 https://github.com/geph-official/geph5/blob/d793aa89e862e11464c46f5cc0ab1b6011215f07/binaries/geph5-client/src/vpn/linux_routing_setup.sh#L1
> 
> Can you give some examples? I'm not quite sure how it should be done.

https://github.com/NixOS/nixpkgs/blob/c02cda63633f02749b3bc95992be02289e03b0b0/pkgs/by-name/op/opensnitch/package.nix#L89

or `substituteInPlace` the shell script.

Reply:
> or substituteInPlace the shell script.

This could be a reference.
https://github.com/NixOS/nixpkgs/blob/c02cda63633f02749b3bc95992be02289e03b0b0/pkgs/by-name/or/orjail/package.nix#L41